### PR TITLE
Graphql types in flow

### PR DESCRIPTION
**Pull request by vincentfretin**

### CODE REVIEW

File: assembl/static2/flow/README [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+    assembl-pshell local.ini
+    from assembl.graphql.schema import generate_schema_json; generate_schema_json()
+    ctr+d
+    cd assembl/static2
+    yarn add apollo-codegen
+    ./node_modules/.bin/apollo-codegen generate js/app/graphql/*.graphql js/app/graphql/mutations/*.graphql --schema /tmp/schema.json --target flow --output flow/graphql_types.flow.js
+    git checkout package.json yarn.lock
+    rm -rf node_modules/
+    yarn
```
Comment (vincentfretin) [inline assembl/static2/flow/README]: I install apollo-codegen only to generate the file, then I restart fresh without it. I have lots of flow errors if I keep it around, probably a flow version mismatch...

File: assembl/graphql/schema.py [hunk 2]
```diff
@@ -2003,11 +2003,11 @@ class Mutations(graphene.ObjectType):
 Schema = graphene.Schema(query=Query, mutation=Mutations)
 
 
-def print_schema_json(schema):
+def generate_schema_json():
```
Comment (cedricmessiant) [inline assembl/graphql/schema.py]: I imagine it was necessary for apollo-codegen? I don't really like the fact that it takes `Schema` from outside ;-)

**Comment by cedricmessiant** [comment]
this is awesome! :1st_place_medal: 
So we have now to replace types we have written by those in `graphql_types.flow.js`, is that it ?

**Comment by vincentfretin** [comment]
Yes

File: assembl/graphql/schema.py (hunk 2 above)
Comment (vincentfretin) [reply assembl/graphql/schema.py]: Not necessary. It just simplify the API. You can see the schéma.py module as a facade, and Schema as internal too.

Comment (vincentfretin) [reply assembl/graphql/schema.py]: I added a generate_schema_json_from_schema(schema, output) api

**Pull request merged**